scripts/process/process.discharge_sparks.R

```diff
@@ -0,0 +1,102 @@
+grab_spark <- function(vals){
+  
+  xs <- seq_len(length(vals))
+  xs <- c(head(xs, 1), xs, tail(xs, 1))
+  ys <- c(min(vals), vals, min(vals))
+  x = svglite::xmlSVG({
+    par(omi=c(0,0,0,0), mai=c(0,0,0,0))
+    plot(xs, ys, type='l', axes=F, ann=F)
+  }, height=0.4, width=2)
+  xml2::xml_attr(xml2::xml_find_first(x, '//*[local-name()="polyline"]'),'points')
+}
+
+
+# Should rename at some point. Not always going to be discharge.
+process.discharge_sparks <- function(viz = as.viz('discharge-sparks')){
```

Comment:
These two need some comment as to what they are for? This one should be called "gage-sparks" maybe? We area actually creating two sets of sparklines for the two styles? @jread-usgs did this toward the end of Irma so I'm not 100% sure what the story is.